library/psa_crypto_storage.h

```diff
@@ -88,22 +83,19 @@ int psa_is_key_present_in_storage( const psa_key_file_id_t key );
  * already occupied non-persistent key, as well as validating the key data.
  *
  *
- * \param key           Persistent identifier of the key to be stored. This
- *                      should be an unoccupied storage location.
- * \param type          Key type (a \c PSA_KEY_TYPE_XXX value).
- * \param[in] policy    The key policy to save.
- * \param[in] data      Buffer containing the key data.
- * \param data_length   The number of bytes that make up the key data.
+ * \param[in] attributes    The attributes of the key to save.
+ *                          The key identifier field in the attributes
+ *                          determines the key's location.
+ * \param[in] data          Buffer containing the key data.
```

Comment:
As far as the storage subsystem is concerned, it's just data. I've actively avoided having it care what this data is. It is somewhat inconvenient for the overall design (https://github.com/gilles-peskine-arm/mbed-crypto/commit/a0081bb3a0a7a423f3f6b6576227a013bf7d714f (part of https://github.com/ARMmbed/mbed-crypto/pull/183) is not very nice), but I'll leave that job for a future PR.

Reply:
We should call it data instead of key data, then.

Reply:
“Key data” is ambiguous, which actually works here. In the PSA code and documentation, we normally use “key material” for the representation of the key itself as opposed to its metadata.

Reply:
I did not pick up on this distinction between "key data" and "key material", believing that "key data" meant key material, if that was the intent. This would be more readable as just "data".